custom/opm/reports.py

```diff
@@ -579,6 +579,11 @@ def get_rows(self):
     @property
     @memoized
     def cases(self):
+        if 'debug_case' in self.request.GET:
```

Comment:
Yeah, hoping this will make it easier to debug some of this stuff in the future.